image/oci_test.go

```diff
@@ -29,36 +29,36 @@ func manifestOCI1FromFixture(t *testing.T, src types.ImageSource, fixture string
 }
 
 func manifestOCI1FromComponentsLikeFixture(configBlob []byte) genericManifest {
-	return manifestOCI1FromComponents(descriptor{
+	return manifestOCI1FromComponents(descriptorOCI1{descriptor: descriptor{
```

Comment:
It would be nice to have tests for the new `annotations` fields, by adding them to `fixtures/oci1.json`, this function, and whatever else needs to know about annotations (probably `TestManifestOCI1UpdatedImage`).  Non-blocking I guess.